rename_test.sh

```diff
@@ -0,0 +1,29 @@
+#! /usr/bin/env bash
+
+find_name(){
+  find test -name "*[\\/:\*\?\"<>\|]*" -o -name "*."
+}
+
+main() {
+  local filename
+  local new_filename
+  while read -r filename; do
+    new_filename=$(echo "$filename" | sed -r \
+      -e "s/\"/'/g" \
+      -e 's/</\[/g' \
```

Comment:
I’m concerned about this one - there’s a meaningful difference here. `<foo>` is both convention for a replacement in a way `[foo]` is not, and also, in command line notation, one means a required argument and the other an optional one.

is there another character that might work that looks visually similar to `< >`, or are we limited to ascii?

Reply:
Yes I dont like much to have to put [] instead of <> for the meaning. I ll check if I can find something that look more similar.

Reply:
I guess I found a solution :D
What do you think now ?